yandex/resource_yandex_mdb_postgresql_cluster_test.go

```diff
@@ -311,6 +311,9 @@ func testAccCheckMDBPGClusterHasUsers(r string, perms map[string][]string) resou
 var defaultUserSettings = map[string]interface{}{
 	"conn_limit": int64(50),
 }
+var testAccMDBPGClusterConfigUpdatedCheckConnLimitMap = map[string]int64{
```

Comment:
Is it the expected value in the test? It's not clear from the name :)

Reply:
test function rename testAccCheckUnmodifiedUserSettings -> testAccCheckConnLimitUpdateUserSettings